src/qibo/symbols.py

```diff
@@ -15,11 +15,11 @@ class Symbol(sympy.Symbol):
             symbolic expressions.
     """
 
-    def __new__(cls, q, matrix=None, name="Symbol"):
+    def __new__(cls, q, matrix=None, name="Symbol", commutative=False):
```

Comment:
Please document the new commutative argument. Ideally, would be really nice to have some example here or in the docs.